smithy-jsonschema/src/main/java/software/amazon/smithy/jsonschema/JsonSchemaVersion.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: Wrong year. [This](https://github.com/awslabs/smithy/pull/1614) updated checkstyle so we can use the short-form license header without the year now:

```
/*
 * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 * SPDX-License-Identifier: Apache-2.0
 */
```

Reply:
updated in cr 3f12c5b.